zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -92,6 +102,28 @@ impl<F: Field> Config<F> {
             push_rindex_inv,
         );
 
+        // Does the current row have bytecode field tag == Length?
+        let is_row_tag_length = |meta: &mut VirtualCells<F>| {
+            let row_tag = meta.query_advice(tag, Rotation::cur());
+            row_tag.expr() * (BytecodeFieldTag::Byte.expr() - row_tag.expr())
+        };
+
+        // Does the current row have bytecode field tag == Byte?
+        let is_row_tag_byte = |meta: &mut VirtualCells<F>| {
+            let row_tag = meta.query_advice(tag, Rotation::cur());
+            row_tag.expr()
+                * (row_tag.expr() - BytecodeFieldTag::Length.expr())
+                * F::from(2).invert().unwrap()
+        };
+
+        // For a row tagged Length, is the length (value) zero?
+        let length_is_zero = IsZeroChip::configure(
```

Comment:
could the `length` of code be 0?

Reply:
Yes,  based on the discussion in https://github.com/appliedzkp/zkevm-specs/issues/102:

An empty code will be represented as:
```
(EMPTY_HASH, TagLength, 0, 0, 0)
```